lib/polo/sql_translator.rb

```diff
@@ -30,10 +31,6 @@ def to_sql
       end
     end
 
-    def records
-      Array.wrap(@record)
```

Comment:
`Array.wrap` doesn't work on an Enumerator, however as far as I can see other than in this test suite, SqlTranslator only ever receives an Enumerable anyway, so I updated the tests and removed this path